lib/full-site-editing/templates-utils.php

```diff
@@ -60,11 +60,7 @@ function gutenberg_render_templates_lists_custom_column( $column_name, $post_id
 		$themes        = array();
 		$is_file_based = false;
 		foreach ( $terms as $term ) {
-			if ( '_wp_file_based' === $term->slug ) {
-				$is_file_based = true;
-			} else {
-				$themes[] = esc_html( wp_get_theme( $term->slug ) );
-			}
+			$themes[] = esc_html( wp_get_theme( $term->slug ) );
 		}
 		echo implode( '<br />', $themes );
 		if ( $is_file_based ) {
```

Comment:
`$is_file_based` is always going to be `false` now (per [R61](https://github.com/WordPress/gutenberg/pull/27910/files#diff-63fe5f44a78ce9690fa091e62f55db710bc89d2c6a3da88dd889a6fa374817acR61)), so we can remove this block.